source/agora/test/SlashingMisbehavingValidator.d

```diff
@@ -63,14 +65,15 @@ private class MissingPreImageEM : EnrollmentManager
 // `MissingPreImageEM` class
 private class NoPreImageVN : TestValidatorNode
 {
-    public static shared UTXOSet utxo_set;
+    public __gshared UTXOSet utxo_set;
```

Comment:
why '__gshared'? Using shared would have required us to define all methods in UTXOSet as 'shared'